### PR TITLE
Add trusted-certificate-issuer helm chart

### PR DESCRIPTION
**Pull request by poussa**

### CODE REVIEW

File: charts/trusted-certificate-issuer/templates/secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  sopin: V0lwbUJCybc2Oc6M06Vz
```
Comment (avalluri) [inline charts/trusted-certificate-issuer/templates/secrets.yaml:3]: I think we should make token pins configurable.

File: charts/trusted-certificate-issuer/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+# Trusted Certificate Issuer Helm chart
+
+Trusted Certificate Service (TCS) is a K8s service to protect signing keys using Intel's SGX technology. Kubernetes certificate signing request (CSR) CSR and cert-manager CertificateReuest APIs are both supported.
```
[the remark this answers is not in the document]
Comment (poussa) [reply charts/trusted-certificate-issuer/README.md]: done.

File: charts/trusted-certificate-issuer/templates/secrets.yaml (hunk 1 above)
Comment (poussa) [reply charts/trusted-certificate-issuer/templates/secrets.yaml]: done

**Comment by poussa** [comment]
@avalluri PTAL now.

File: charts/trusted-certificate-issuer/templates/secrets.yaml (hunk 1 above)
Comment (avalluri) [reply charts/trusted-certificate-issuer/templates/secrets.yaml]: > done

Thanks.  We should also list these new params in the ReadME.

Comment (poussa) [reply charts/trusted-certificate-issuer/templates/secrets.yaml]: added

**Comment by poussa** [comment]
@avalluri PTAL

**Review by avalluri — APPROVED** [review]
Looks good to me.

**Pull request merged**